src/main.js

```diff
@@ -1,18 +1,29 @@
 import Car from "./Car.js";
+import RacingGame from "./RacingGame.js";
 import {
   createReadlineInterface,
   getCarNamesFromInput,
+  getTrialCountFromInput,
   printRacingGameResult,
 } from "./io.js";
-import { racingGameGenerator } from "./racing.js";
+
+const getCanMove = () => {
+  const MIN = 0;
+  const MAX = 9;
+  const randomNumber = Math.floor(Math.random() * (MAX - MIN + 1)) + MIN;
+
+  return randomNumber >= 4;
+};
 
 const main = async () => {
   const readline = createReadlineInterface();
 
   try {
     const names = await getCarNamesFromInput(readline);
     const cars = names.map((name) => new Car(name));
-    const racingGame = racingGameGenerator(cars);
+    const trialCount = await getTrialCountFromInput(readline);
+
+    const racingGame = new RacingGame(cars, trialCount, getCanMove);
```

Comment:
```suggestion
    const racingGame = new RacingGame(cars, 0, getCanMove);
```
이렇게하면 어떻게 될까요!?
혹은 어떻게 처리되어야 좋을까요?